src/main/java/seedu/foodrem/commons/core/index/Index.java

```diff
@@ -45,10 +37,18 @@ public static Index fromOneBased(int oneBasedIndex) {
         return new Index(oneBasedIndex - 1);
     }
 
+    public int getZeroBased() {
+        return zeroBasedIndex;
+    }
+
+    public int getOneBased() {
+        return zeroBasedIndex + 1;
+    }
+
     @Override
     public boolean equals(Object other) {
         return other == this // short circuit if same object
-                || (other instanceof Index // instanceof handles nulls
-                && zeroBasedIndex == ((Index) other).zeroBasedIndex); // state check
+            || (other instanceof Index // instanceof handles nulls
+            && zeroBasedIndex == ((Index) other).zeroBasedIndex); // state check
```

Comment:
The auto formatting settings seem off. The style guide for cs2103t specifies 8 spaces for line breaks, not 4.